modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -642,11 +643,11 @@ private CompletableFuture<Void> triggerTxRecovery(UUID txId, UUID senderId) {
         HybridTimestamp opStartTs;
 
         if (request instanceof ReadWriteReplicaRequest) {
-            opStartTs = clockService.now();
+            opStartTs = clockService.current();
```

Comment:
It seems that now we only trigger ticking with `HLC.now()` (on the transaction operations processing path) in the following cases:

1. Tx begin
2. Write operation in a tx
3. Commit

Is this true?

In any case, could you please write a comment somewhere that would define the principle that is used to choose when we use `now()` and when `current()`?